<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Karthisri" failures="2" tests="2" name="Test" time="0.078" errors="0" timestamp="2022-03-28T19:08:35 IST">
  <testcase classname="Maven.Karthi.Sample_RetryCls" name="UsernameValidation" time="0.014">
    <failure type="java.lang.AssertionError" message="expected [Hi] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hi] but found [Hello]
at Maven.Karthi.Sample_RetryCls.UsernameValidation(Sample_RetryCls.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- UsernameValidation -->
  <testcase classname="Maven.Karthi.Sample_RetryCls" name="passwordValidation" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [Sri] but found [Karthi]">
      <![CDATA[java.lang.AssertionError: expected [Sri] but found [Karthi]
at Maven.Karthi.Sample_RetryCls.passwordValidation(Sample_RetryCls.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- passwordValidation -->
</testsuite> <!-- Test -->
